dockerfiles/Dockerfile.openvino-centos7

```diff
@@ -0,0 +1,105 @@
+    cd $MY_ROOT && \
+# libusb1.0.22
+    cd /opt/ && wget https://github.com/libusb/libusb/archive/v1.0.22.zip && \
+    unzip v1.0.22.zip && rm -rf v1.0.22.zip && cd  /opt/libusb-1.0.22 && \
+# bootstrap steps
+    ./bootstrap.sh && \ 
+    ./configure --disable-udev --enable-shared && \
+    make -j4 && \
+# configure libusb1.0.22
+    cd /opt/libusb-1.0.22/libusb && \
+    /bin/mkdir -p '/usr/local/lib' && \
+    /bin/bash ../libtool   --mode=install /usr/bin/install -c   libusb-1.0.la '/usr/local/lib' && \
+    /bin/mkdir -p '/usr/local/include/libusb-1.0' && \
+    /usr/bin/install -c -m 644 libusb.h '/usr/local/include/libusb-1.0' && \
+    /bin/mkdir -p '/usr/local/lib/pkgconfig' && \
+# Install openvino
+    yum-config-manager --add-repo https://yum.repos.intel.com/openvino/2021/setup/intel-openvino-2021.repo && \
+    rpm --import https://yum.repos.intel.com/openvino/2021/setup/RPM-GPG-KEY-INTEL-OPENVINO-2021 && \
+    yum update -y && yum list intel-openvino* && \
+    yum install -y $YUM_OV_PACKAGE && \
+    cd ${INTEL_OPENVINO_DIR}/install_dependencies/ && ./install_openvino_dependencies.sh -y && \
+    printf "\nexport LD_LIBRARY_PATH=\${LD_LIBRARY_PATH}:/usr/local/lib\n" >> /opt/intel/openvino_2021.4.582/bin/setupvars.sh && \
+    cd /opt/libusb-1.0.22 && \
+    /usr/bin/install -c -m 644 libusb-1.0.pc '/usr/local/lib/pkgconfig' && \
+    cp /opt/intel/openvino_2021/deployment_tools/inference_engine/external/97-myriad-usbboot.rules /etc/udev/rules.d/ && \
+    ldconfig && \
+# Install GPU runtime and drivers
+    cd ${MY_ROOT} && \    
+    mkdir /tmp/opencl && \
+    cd /tmp/opencl && \
+    yum install -y epel-release && \
+    yum install -y ocl-icd ocl-icd-devel && \
+    wget -O intel-igc-core-1.0.2597-1.el7.x86_64.rpm https://sourceforge.net/projects/intel-compute-runtime/files/19.41.14441/centos-7/intel-igc-core-1.0.2597-1.el7.x86_64.rpm/download && \
+    wget -O intel-opencl-19.41.14441-1.el7.x86_64.rpm https://sourceforge.net/projects/intel-compute-runtime/files/19.41.14441/centos-7/intel-opencl-19.41.14441-1.el7.x86_64.rpm/download && \
+    wget -O intel-igc-opencl-devel-1.0.2597-1.el7.x86_64.rpm https://sourceforge.net/projects/intel-compute-runtime/files/19.41.14441/centos-7/intel-igc-opencl-devel-1.0.2597-1.el7.x86_64.rpm/download && \
+    wget -O intel-igc-opencl-1.0.2597-1.el7.x86_64.rpm https://sourceforge.net/projects/intel-compute-runtime/files/19.41.14441/centos-7/intel-igc-opencl-1.0.2597-1.el7.x86_64.rpm/download && \
+    wget -O intel-gmmlib-19.3.2-1.el7.x86_64.rpm https://sourceforge.net/projects/intel-compute-runtime/files/19.41.14441/centos-7/intel-gmmlib-19.3.2-1.el7.x86_64.rpm/download && \
+    wget -O intel-gmmlib-devel-19.3.2-1.el7.x86_64.rpm https://sourceforge.net/projects/intel-compute-runtime/files/19.41.14441/centos-7/intel-gmmlib-devel-19.3.2-1.el7.x86_64.rpm/download && \
+    rpm -i /tmp/opencl/*.rpm  && \
+    ldconfig  && \
+    rm -rf /tmp/opencl && \
+# Installing gcc-7
```

Comment:
Could you use gcc-10 instead? gcc-7 doesn't have full C++17 support.

Reply:
@snnn Updated gcc version to 10.
Also validated the new changes to dockerfile by building a image locally and validating all the hardware's we support - CPU, GPU, MYRIADX, VADM inside the container.